appveyor.yml

```diff
@@ -10,10 +10,16 @@ init:
 install:
   ps: Bootstrap
 
+environment:
+  NOT_CRAN: true
+  USE_RTOOLS: true
+  PKGTYPE: both
+
 # Adapt as necessary starting from here
 
 build_script:
   - travis-tool.sh install_deps
+  - travis-tool.sh install_bioc_deps
```

Comment:
This needs to happen before install_deps, the build is failing before it gets here.

Reply:
OK, yeah, that fixes it.

I'm about to run more experiments to determine this empirically, but are all 3 of these lines needed or, if you pick the right ones and the right order, can it be minimized?

```
build_script:
  - travis-tool.sh install_bioc IRanges
  - travis-tool.sh install_bioc_deps
  - travis-tool.sh install_deps
```

Reply:
I think you could probably remove the IRanges one, and just use the `install_bioc_deps()` one.